packages/compartment-mapper/src/policy.js

```diff
@@ -28,7 +30,12 @@ export const ATTENUATORS_COMPARTMENT = '<ATTENUATORS>';
  */
 const selectiveCopy = (from, to, list) => {
   if (!list) {
-    list = keys(from);
+    const descs = getOwnPropertyDescriptors(from);
+    list = ownKeys(from).filter(
+      key =>
+        // @ts-expect-error TypeScript still confused about a symbol as index
+        descs[key].enumerable,
+    );
   }
   for (let index = 0; index < list.length; index += 1) {
```

Comment:
@naugtur Might be worth you time to investigate how a policy would name a symbol to be preserved.